modules/doc/config.yml

```diff
@@ -44,7 +44,7 @@ Extensions:
                 Creating an Application: getting_started/new_users.md
                 Tutorials: getting_started/examples_and_tutorials/index.md
                 Supplemental Resources: getting_started/supplemental_resources/index.md
-                INL HPC Remote Access: help/inl/index.md
+                INL HPC Services: help/inl/index.md
```

Comment:
"Remote Access" no longer encompasses what we are doing in this section

Reply:
I think this is a good change that reflects how broad this page now is - resolving this comment.